terraform/gitops/proxy-pm4ml/pm4ml.tf

```diff
@@ -8,17 +8,23 @@ module "generate_pm4ml_files" {
     pm4ml_release_name                              = each.key
     pm4ml_namespace                                 = each.key
     storage_class_name                              = var.storage_class_name
-    pm4ml_sync_wave                                 = var.pm4ml_sync_wave + index(keys(var.app_var_map), each.key)
+    pm4ml_sync_wave                                 = var.pm4ml_sync_wave
     external_load_balancer_dns                      = var.external_load_balancer_dns
     istio_internal_wildcard_gateway_name            = var.istio_internal_wildcard_gateway_name
     istio_internal_gateway_namespace                = var.istio_internal_gateway_namespace
     istio_external_wildcard_gateway_name            = var.istio_external_wildcard_gateway_name
     istio_external_gateway_namespace                = var.istio_external_gateway_namespace
     pm4ml_wildcard_gateway                          = each.value.pm4ml_ingress_internal_lb ? "internal" : "external"
-    proxy_id                                        = each.value.pm4ml_proxy_id
```

Comment:
I think there is a reason why we have pm4ml_proxy_id separately. There are differences in the allowed pattern for k8s resource names and DFSP IDs in the scheme. So we want them to be separate parameters.

Reply:
Also proxy ids are kind of for internal usage and we can choose proper/suitable names for them.

Reply:
Actually we pass the same ID as DFSPID in the sdk scheme adapter which sends requests to switch with that ID. So its not internal. So lets imagine the switch provided a long DFSP ID and if we use that to deploy the instance, we may endup having some k8s resources exceed the 63 char limit. Which again we need to deal with some nameOverrides. So I think its unnecessary complication, let me know if it makes sense.

Reply:
fixed in 40e38de11c3e7bcfe903f3469b41a0619522306c